### DISCUSSION TITLE
K8s startup, readiness, and liveliness probes?

### DISCUSSION THREAD

**Discussion by uzubair-miovision** [openfga/community#172] [Q&A] [answered]
I can't seem to find the startup, readiness, and liveliness probes for openfga deployed in Kubernetes. Any suggestions/ recommendations?

**Comment by matthewpereira** [comment]
Hey @uzubair-miovision , our resident K8s expert will be able to post a response to this early next week. 

In the meantime, just in case it helps, are you by any chance looking for https://github.com/openfga/helm-charts/blob/cb87e94a1b0e8e125c4d115a8a2d0851ded37ab4/charts/openfga/templates/deployment.yaml#L285C19-L285C19 ?

Or, are you asking for tips on integration and use? :)

`v0.1.5` introduced support for the [GRPC Health Checking Protocol](https://github.com/grpc/grpc/blob/master/doc/health-checking.md). The server's health can be checked with the grpc or HTTP health check endpoints (the `/healthz` endpoint is just a proxy to the grpc health check RPC).

**Comment by jon-whit** [comment]
@uzubair-miovision we do not currently have liveness or startup probes defined in the OpenFGA chart. I've just opened a PR that adds some defaults and allows you to specify custom probes if you desire.

https://github.com/openfga/helm-charts/pull/52

Let me know if that works for you and your use case 👍

**Reply by uzubair-miovision** [reply]
Thanks @matthewpereira and @jon-whit . You guys are amazing.

**Comment by jon-whit** [comment, marked as answer]
@uzubair-miovision `v0.1.21` of the OpenFGA Helm chart should now include this support. Please let us know how it works for ya!

https://artifacthub.io/packages/helm/openfga/openfga/0.1.21
